service/src/main/java/org/cbioportal/service/util/AlterationEnrichmentUtil.java

```diff
@@ -79,7 +79,7 @@ private AlterationEnrichment createAlterationEnrichment(List<? extends Alteratio
         alterationEnrichment.setSet2CountSummary(set2CountSummary);
         alterationEnrichment.setEntrezGeneId(alterationCountByGene.getEntrezGeneId());
         alterationEnrichment.setHugoGeneSymbol(gene.getHugoGeneSymbol());
-        alterationEnrichment.setCytoband(gene.getCytoband());
+        //alterationEnrichment.setCytoband(gene.getCytoband()); //set value at the frontend
```

Comment:
remove commented out line. If you want to capture this information then I would put it in the javadoc comment for this method

Reply:
removed.